packages/components/src/templates/next/components/native/ListItem/ListItem.tsx

```diff
@@ -21,6 +21,7 @@ const ListItem = ({ content, LinkComponent, site }: ListItemProps) => {
             <OrderedList
               key={index}
               {...item}
+              level={!!level ? level + 1 : 1}
```

Comment:
question: why is this `level + 1` and `1` and not `level` and `0`? does this have something to do with what tiptap sends us?

Reply:
Hmm nothing to do with Tiptap here. Cos `level` is optional so the burden of setting the level is not placed on users of a list component, so for the OrderedList and UnorderedList components, they will assume that they are at level 0 if it is not provided. This increment in level is because we are starting on the next nested list, so the first nested list will have a level of 1.

This has no schema change either, purely just a components thing to track which level the list is being called at.

Reply:
ah ok wait let me double check - this is an ordered list inside another ordered list so the level has to be minimally 1?

missed this on initial read

Reply:
Oops missed this, the first list is always at level 0 (zero-based indexing), then we increment the level every time we start a new nested list.